assets/components/patronsEvents/patronsEvents.jsx

```diff
@@ -25,13 +24,14 @@ export default function PatronsEvents(props: PropTypes) {
     <PageSection heading="Other ways you can support us" modifierClass="patrons-events">
       <OtherProduct
         modifierClass="patrons"
-        gridImg="newsroom"
-        imgAlt="newsroom"
+        gridImg="windrushGreyscale"
+        imgAlt="passengers aboard windrush ship"
         heading="Patrons"
-        copy="The Patron tier is for those who want a deeper relationship with the Guardian and its journalists"
+        copy="Support from our patrons is crucial to ensure that generations to come will be able to enjoy The Guardian."
```

Comment:
Sorry to be a pedant but I think this should be **P**atrons

Reply:
Ahhh thanks will fix

Reply:
I will not update the screenshots again though 😛